apps/web/src/app/[lang]/project/[slug]/page.tsx

```diff
@@ -0,0 +1,19 @@
+import ProjectOverview from '@/app/[lang]/main/_components/ProjectOverview';
+import HydratePage from '@/components/basic/HydratePage';
+import { getGetProjectListQueryOptions } from '@/models/project-read-api/project-read-api';
+import getGetProjectDetailQueryKey from '@/utils/query/getQueryClient';
+
+const ProjectPage = async () => {
+  const queryClient = getGetProjectDetailQueryKey();
```

Comment:
```suggestion
  const queryClient = getQueryClient();
```
queryClient를 선언하는 부분입니다. 코드자체가 잘못되었습니다.

Reply:
넵 확인했습니다.

<img width="277" alt="스크린샷 2024-10-29 오전 2 47 48" src="https://github.com/user-attachments/assets/d92b872c-6d56-4f4d-97d7-ca3f41916e04">

구조는 이렇게 잡으면 될까요?

Reply:
지금 왜 `/overview`,`/overview/[slug]`  페이지를 두개 만들고 있는건가요?

```typescript
overview
  [slug]
     page.tsx
     view..tsx
     hooks/
     _components/     
```
이런식으로 내가 만드는 페이지안에서만 폴더랑 만들어서 처리하라는 의미입니다.